.tool-versions

```diff
@@ -1,5 +1,6 @@
+
 nodejs 20.18.2
-python 3.8.19
+python 3.9
```

Comment:
Usually, this needs to be a specific version (e.g. `3.9.21`)

Reply:
Used 3.9.21 as you suggested